feature-changeaddress/src/main/kotlin/com/hedvig/android/feature/changeaddress/ui/ChangeAddressOfferDestination.kt

```diff
@@ -89,194 +77,57 @@ internal fun ChangeAddressOfferDestination(
-              style = MaterialTheme.typography.bodyLarge,
-            )
-            Text(
-              text = uiState.numberCoInsured.input.toString(),
-              style = MaterialTheme.typography.bodyLarge,
-              color = MaterialTheme.colorScheme.secondary,
-            )
-          }
-          Divider(thickness = 1.dp, modifier = Modifier.padding(horizontal = 16.dp))
-
-          Row(
-            horizontalArrangement = Arrangement.SpaceBetween,
-            modifier = Modifier
-              .padding(horizontal = 16.dp, vertical = 21.dp)
-              .fillMaxWidth(),
-          ) {
-            Text(
-              text = stringResource(id = hedvig.resources.R.string.CHANGE_ADDRESS_HOUSING_TYPE_LABEL),
-              style = MaterialTheme.typography.bodyLarge,
-            )
-            Text(
-              text = uiState.housingType.input?.displayNameResource()?.let {
-                stringResource(id = it)
-              } ?: "-",
-              style = MaterialTheme.typography.bodyLarge,
-              color = MaterialTheme.colorScheme.secondary,
-            )
-          }
-          Divider(thickness = 1.dp, modifier = Modifier.padding(horizontal = 16.dp))
-
-          Row(
-            horizontalArrangement = Arrangement.SpaceBetween,
-            modifier = Modifier
-              .padding(horizontal = 16.dp, vertical = 21.dp)
-              .fillMaxWidth(),
-          ) {
-            Text(
-              text = stringResource(id = hedvig.resources.R.string.CHANGE_ADDRESS_NEW_LIVING_SPACE_LABEL),
-              style = MaterialTheme.typography.bodyLarge,
-            )
-            Text(
-              text = uiState.squareMeters.input ?: "-",
-              style = MaterialTheme.typography.bodyLarge,
-              color = MaterialTheme.colorScheme.secondary,
-            )
-          }
-        }
-      }
+        Text(
+          text = "Hittar du inte det du söker?",
+        )
 
-      Spacer(modifier = Modifier.padding(top = 8.dp))
+        Spacer(modifier = Modifier.padding(top = 24.dp))
 
-      TextButton(
-        onClick = { navigateBack() },
-        modifier = Modifier.align(CenterHorizontally),
-      ) {
         Text(
-          text = stringResource(id = hedvig.resources.R.string.EDIT_BUTTON_LABEL),
-          color = MaterialTheme.colorScheme.tertiary,
-          style = MaterialTheme.typography.bodyLarge,
+          text = stringResource(id = hedvig.resources.R.string.open_chat),
+          color = MaterialTheme.colorScheme.onPrimary,
+          modifier = Modifier
+            .padding(
+              horizontal = 16.dp,
+              vertical = 8.dp,
+            )
+            .clip(SquircleShape)
+            .clickable { openChat() }
+            .background(shape = SquircleShape, color = MaterialTheme.colorScheme.primary)
+            .padding(
+              horizontal = 16.dp,
+              vertical = 8.dp,
+            ),
         )
```

Comment:
Might wanna use `androidx.compose.material3.TextButton` if we want a text which is also clickable? That way you get the slot for the shape, automatically get the ripple to follow that shape, and other guarantees like the touch size being big enough and so on. Would it fit in this case?